hail/python/hail/expr/aggregators/__init__.py

```diff
@@ -35,5 +35,6 @@
     'group_by',
     'downsample',
     'array_agg',
-    '_prev_nonnull'
+    '_prev_nonnull',
+    '_impute_type'
```

Comment:
Nit: Why do we not use trailing commas here? It makes diffs like these smaller.

Reply:
yeah, I usually to add/leave them, just messed up here